app/components/metadata-panel.tsx

```diff
@@ -470,6 +478,37 @@ const TabPanelData = ({ dimensions }: { dimensions: Component[] }) => {
     return dimensions.map((d) => ({ label: d.label, value: d }));
   }, [dimensions]);
 
+  const locale = useLocale();
+  const grouped = groupBy(
+    dimensions.flatMap((d): Component[] => {
+      if ("originalIris" in d) {
+        return (
+          d.originalIris?.map((o) => ({
```

Comment:
Nit: do we need `?` here, when `originalIris` property exists?

Reply:
The property exists but could be undefined ;)

Reply:
I moved the check to the if ;)